capability/CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Changelog
+This file documents all notable changes made to this project since the initial fork.
```

Comment:
The fork history should be explained

Reply:
I have added a link to the original repo here. The move to moby/sys will be explained in 0.2.0 changelog entry.